Index: Chirp.CLI.Client/UserInterface.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>namespace Chirp.CLI;\r\n\r\nstatic class UserInterface\r\n{\r\n    \r\n}
===================================================================
diff --git a/Chirp.CLI.Client/UserInterface.cs b/Chirp.CLI.Client/UserInterface.cs
--- a/Chirp.CLI.Client/UserInterface.cs	(revision 0d8b5d77780f3dcbeba5965fb0370afc9e1fb878)
+++ b/Chirp.CLI.Client/UserInterface.cs	(date 1725455371248)
@@ -1,6 +1,14 @@
+using System.Runtime.CompilerServices;
+
 namespace Chirp.CLI;
-
+using CsvHelper;
 static class UserInterface
 {
-    
+    static void printCheeps(IEnumerable<Cheep> cheeps)
+    {
+        foreach (var cheep in cheeps)
+        {
+            Console.WriteLine(cheep);
+        }
+    }
 }
\ No newline at end of file
